code/game/objects/structures/trash_pile.dm

```diff
@@ -219,7 +219,7 @@
 	return I
 
 /obj/structure/trash_pile/proc/produce_beta_item()
-	var/path = pick(prob(6);/obj/item/weapon/storage/pill_bottle/tramadol,
+	var/path = pick(prob(6);/obj/item/weapon/storage/pill_bottle/paracetamol,
```

Comment:
`//VOREStation Edit`

Reply:
The entire file is a vorestation-specific file (which is why there are no //vorestation edits in it. However, I am going to append _vr to the file so that it is more obviously marked as a vorestation-specific file